src/Og.php

```diff
@@ -59,4 +190,79 @@ public static function groupManager() {
     return \Drupal::service('og.group.manager');
   }
 
+  /**
+   * Invalidate cache.
+   *
+   * @param $gids
+   *   Array with group IDs that their cache should be invalidated.
+   */
+  public static function invalidateCache($gids = array()) {
+    // @todo We should not be using drupal_static() review and remove.
```

Comment:
yeah, I saw a few uses in node.module so assumed it's still ok. So where do you hold it - simply in a protected var?